extractor/src/main/java/org/schabi/newpipe/extractor/services/youtube/linkHandler/YoutubeChannelLinkHandlerFactory.java

```diff
@@ -49,6 +49,17 @@ public String getUrl(String id, List<String> contentFilters, String searchFilter
         return "https://www.youtube.com/" + id;
     }
 
+    /**
+     * Returns true if path conform to
+     * custom short channel urls like youtube.com/yourcustomname
+     *
+     * @param splitPath path segments array
+     * @return true - if value conform to short channel url, false - not
+     */
+    public boolean isCustomShortChannelUrl(String[] splitPath) {
+        return splitPath.length == 1 && !splitPath[0].matches("playlist|watch");
```

Comment:
http://www.youtube.com/attribution_link?a=JdfC0C9V6ZI&u=%2Fwatch%3Fv%3DEhxJLojIE_o%26feature%3Dshare is a stream URL, but is accepted by your regex.

Reply:
Hi. Thank you for review. Do you think that simply adding "attribution_link" to exlude those links with regex would be sufficent solution ? I couldn't come up with something better that that.

Reply:
Mmmh, probably a list of all pages not pointing to a channel should be added to that regex

Reply:
Other YouTube URLs:
- https://m.youtube.com/select_site

- https://www.youtube.com/?app=desktop&persist_app=1

@Stypox what do you mean exactly?